crm_location/README.rst

```diff
@@ -0,0 +1,13 @@
+CRM location
+============
+
+This module introduces a better zip.
+
+It enables zip, city, state and country auto-completion on lead and companies.
```

Comment:
Creation section `Installation` and mention that this module depends on module _base_location_ that it's located on OCA/partner-contact repository.